vpr/src/base/read_route.cpp

```diff
@@ -259,12 +262,26 @@ static void process_nodes(std::ifstream& fp, ClusterNetId inet, const char* file
                               "The coordinates of node %d does not match the rr graph", inode);
                 }
                 offset = 2;
+
```

Comment:
Looks like these changes are coming from another commit unrelated to printing the segment name.

Can you adjust the branch so it only contains the segment name commit?

Reply:
I messed up my forks tree a bit so these commits relate to: https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1318 . Once that pull request goes through this should clear up, sorry for the mess!